<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgOptions</class>
 <widget class="QDialog" name="dlgOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>402</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>417</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Startup</string>
   </property>
   <widget class="QCheckBox" name="chbAutoCardReading">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>determins whether the application should start reading the card from the moment it's detected in the cartdreader or not</string>
    </property>
    <property name="text">
     <string>Automatic card reading</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbWinAutoStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Determins whether this application should start when windows starts or not</string>
    </property>
    <property name="text">
     <string>Activate on Windows startup</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbStartMinimized">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Determins whether the mainwindow should stay hidden on startup or not (Tray Icon will stay available)</string>
    </property>
    <property name="text">
     <string>Start minimized</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Certificates</string>
   </property>
   <widget class="QCheckBox" name="chbRegCert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Register certificates on insert</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbRemoveCert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove certificates on close</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Layout</string>
   </property>
   <widget class="QCheckBox" name="chbShowNotification">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Show notification</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Card reader</string>
   </property>
   <widget class="QComboBox" name="cmbCardReader">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>77</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbShowPicture">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>261</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Show picture</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbShowToolbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>261</width>
     <height>19</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Determins whether the toolbar gets shown or not</string>
   </property>
   <property name="text">
    <string>Show toolbar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>dlgOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
